2/unrm

```diff
@@ -20,8 +20,7 @@ cd $archivedir
 # If given no arguments, just show a listing of the deleted files
 if [ $# -eq 0 ] ; then
   echo "Contents of your deleted files archive (sorted by date):"
-  ls -FC | sed -e 's/\([[:digit:]][[:digit:]]\.\)\{5\}//g' \
-    -e 's/^/  /' 
```

Comment:
You mean the `-d 's/^/ /'`, right? Yea, this is because I change ls -FC to ls -F, so ls output won't be in a line any more, which can be used by uniq later, since we don't print the output in one line, I think we don't need to add extra heading space. 

But I could misunderstand the regex purpose.